test/parallel/test-crypto-rsa-dsa.js

```diff
@@ -184,6 +187,88 @@ test_rsa('RSA_NO_PADDING');
 test_rsa('RSA_PKCS1_PADDING');
 test_rsa('RSA_PKCS1_OAEP_PADDING');
 
+// Test OAEP with different hash functions.
+test_rsa('RSA_PKCS1_OAEP_PADDING', undefined, 'sha1');
+test_rsa('RSA_PKCS1_OAEP_PADDING', 'sha1', undefined);
+test_rsa('RSA_PKCS1_OAEP_PADDING', 'sha256', 'sha256');
+test_rsa('RSA_PKCS1_OAEP_PADDING', 'sha512', 'sha512');
```

Comment:
would an explicit `'sha512', 'sha256')` test be worth adding?

Reply:
Addressed in e624eb0fc54618a0cafcd6e9c77bc8df318679aa.